src/load-default-blueprint-from-disk.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const path = require('path');
-const getProjectOptions = require('./get-project-options');
+const { getProjectOptions } = require('./get-project-options');
```

Comment:
It looks like this can go back to `const getProjectOptions = require('./get-project-options');` now?